carrow.go

```diff
@@ -9,7 +9,9 @@ import (
 
 /*
 #cgo pkg-config: arrow plasma
-#cgo LDFLAGS: -lcarrow -L.
+#cgo LDFLAGS: -lcarrow
+#cgo linux LDFLAGS: -L./bindings/linux-x86_64
+#cgo darwin LDFLAGS: -L./bindings/darwin-x86_64
```

Comment:
Since we don't have darwin build - remove & create issue

Reply:
#44